classes/Path.php

```diff
@@ -80,8 +75,6 @@ public function cacheTwigPath(): string
 
     /**
      * Get the HTML Purifier cache path.
```

Comment:
Maybe we can remove the entire docblock here (and elsewhere) as well?

They just repeat what is already obvious from the method name, and now that we have the return type declarations, they don't really add any value (only cost, because they need to be maintained). What do you think?